### PR TITLE
Msw 93240698 finish modal button

### PR DESCRIPTION
**Pull request by michaelwehrley**
Modal Displays Restart Message

* The alert modal restarts the user's session.
* Message gives the user the chance to restart.

[Finishes: #93240698]

### CODE REVIEW

File: test/unit/services/CurrentUserSpec.js [hunk 1]
```diff
@@ -65,6 +65,28 @@ describe('CurrentUser', function() {
...
+    });
   });
 
+  describe('#isFinished', function() {
+    beforeEach(function() {
+      categoryConstants.COMPLETED_COUNT = 0;
+      CurrentUser.initialize();
+    });
+
+    it('returns \'false\' if number of categories completed is not equal to the COMPLETED COUNT', function() {
```
Comment (ericcf) [inline test/unit/services/CurrentUserSpec.js]: I think either the titles or the expectations for these last two specs are mixed up?

Comment (michaelwehrley) [inline test/unit/services/CurrentUserSpec.js]: Thanks.

**Comment by michaelwehrley** [comment]
This should be all set.

**Comment by ericcf** [comment]
:+1: :zap:

**Pull request merged**